modules/example/App.tsx

```diff
@@ -1,16 +1,40 @@
 import * as React from "react";
-import { HandshakeStatus } from "@replit/extensions";
+import {
+  HandshakeStatus,
+  exec,
+  messages,
+  extensionPort,
+  proxy,
+  ExecResult,
+} from "@replit/extensions";
 import { useReplit } from "@replit/extensions-react";
 import "./App.css";
 
 export default function App() {
   const { status, error, filePath, replit } = useReplit();
 
-  if (typeof window !== 'undefined') {
+  if (typeof window !== "undefined") {
     // @ts-ignore
     window.replit = replit;
   }
 
+  const execute = async () => {
+    const dispose = await exec({
```

Comment:
actually you know what, i really don't like `await` here referring to the start of execution rather than the end